unittests/API/NodeAPITest.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <gtest/gtest.h>
+#include <hermes/hermes.h>
+#include <hermes/VM/BuildMetadata.h>
+#include <cstdint>
+
+#include "js_native_api.h"
+
+using namespace facebook::hermes;
+using namespace facebook::jsi;
+
+namespace {
+
+class NodeAPITest : public ::testing::Test {
+protected:
+  // Runs once before all tests in this test suite
+  void SetUp() {
+    // The metadata aren't shared across libraries, so we need to build first
+    ::hermes::vm::buildMetadataTable();
```

Comment:
I get segmentation faults without this. I don't know if we can expect this to have been called or we should move this into `hermesNodeApi`'s `createNodeApiEnv` implementation 🤔

Reply:
It is caused by an issue in the "createNodeApiEnv": `return &res.getValue();`.
It must be `return res.getValue();`

Reply:
🤦 thanks.